### PR TITLE
Added 3Js HindiCourse

### PR DESCRIPTION
**Pull request by subhayudutta**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
ThreeJs

### Why is this valuable (or not)?
It is valuable as threejs is trending now a days.

### How do we know it's really free?
It is free available in youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course/tutorial.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
Kindly refrain from using shortened links; please use the full link. We prefer "playlist" links for YouTube. Also, replace "|" with "-" in future contributions.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -633,6 +633,13 @@
 * [System Design Series By Coding Ninjas](https://www.youtube.com/playlist?list=PLrk5tgtnMN6RvXrfflstJWgcPFQ_vTOV9) - Coding Ninjas
 
 
+### Threejs
+
+* [Three.js Course For Beginners | Hindi Tutorial](https://youtu.be/d-LVbNpz4A4?si=g3Tcs5quw6HPk99X) - Fun Coding
+* [Three.js Crash Course For Beginners | Create This Awesome 3D Website!](https://youtu.be/_OwJV2xL8M8?si=DohxEgr0osuVndm8) - Developedbyed
+* [Threejs by Amirul Islam - Hindi](https://youtube.com/playlist?list=PLjXsIhZZY9KJP9cFq9NqxTucfFCnVzRUn&si=IO3x9fajJ1mHcYnM) - Amirul Islam
```
Comment (sohansai) [inline courses/free-courses-hi.md:640]: The playlist contains diverse videos from various authors, and it's in English; please consider contributing to language-specific free courses next time.

**Comment by subhayudutta** [comment]
@sohansai I have made the following changes as you suggested. Please check.

**Comment by eshellman** [comment]
as this is a collection of videos rather than a course, we won't include it in our list

**Pull request closed without merging**